tests/queries/0_stateless/02011_tuple_vector_functions.sql

```diff
@@ -0,0 +1,58 @@
+SELECT tupleHammingDistance(tuple(1), tuple(1));
+SELECT tupleHammingDistance((1, 3), (1, 2));
+
+SELECT (1, 2) + tupleMultiply((3, 4), (5, 1)) - (6, 3);
```

Comment:
All tests in file uses contants, it's good to test with non constant column (adding `materialize` or/and using `numbers(n)` table function or table `system.numbers`/`system.numbers_mt`). 
Actually, because we use `useDefaultImplementationForConstants` behavior  is the same. But it can be changed later and couple of more complex cases don't hurt in any case.

Reply:
Will do it.